### PR TITLE
build: use @stoplight/scripts for build & release

### PR DESCRIPTION
**Pull request by P0lip**
https://app.circleci.com/pipelines/github/stoplightio/json-ref-readers/40/workflows/ddd637ab-0d55-4078-95f4-d47ec69c93f0/jobs/58 release step fails due to the auth problem. Hopefully using scripts solves this issue.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "extends": "@stoplight/scripts/tsconfig.json",
+
+  // target all ts files
   "include": ["."],
-  "exclude": ["**/__*__/**"],
+
   "compilerOptions": {
-    "outDir": "dist",
-    "module": "commonjs"
```
Comment (wmhilton) [inline tsconfig.json:7]: Is this a breaking change?

Comment (wmhilton) [reply tsconfig.json:7]: I think it is, because the default for `@stoplight/scripts` is `"module": "esnext",`

Comment (P0lip) [reply tsconfig.json:7]: Done, reverted

**Review by billiegoose — APPROVED** [review]
LGTM

**Pull request merged**